extensions/wikia/AdEngine/js/slot/resolvedState.js

```diff
@@ -0,0 +1,72 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.resolvedState', [
+	'wikia.log',
+	'wikia.querystring'
+], function (log, QueryString) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.slot.resolvedState',
+		qs = new QueryString();
+
+	function getQueryParam() {
+		return qs.getVal('resolved_state', null);
+	}
+
+	function isForcedByURLParam() {
+		return [true, 'true', '1'].indexOf(getQueryParam()) > -1;
+	}
+
+	function isBlockedByURLParam() {
+		return [false, 'blocked', 'false', '0'].indexOf(getQueryParam()) > -1;
+	}
+
+	function paramsAreCorrect (params) {
+		var correctSplitImages = params.resolvedState.leftImageSrc && params.resolvedState.rightImageSrc;
```

Comment:
Just asking, didn't we want to resign from left/right naming? (use first, second, and give parameter which of those will have video overlay)